modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -604,4 +608,200 @@ private void deleteApplication(String applicationId) throws Exception {
+        after = getLink(firstPageLinks, "next");
+        before = getLink(firstPageLinks, "previous");
+
+        Assert.assertNotNull(after, "After value should not be null on the first page.");
+        Assert.assertNull(before, "Before value should be null on the first page.");
+
+        // Step 2: Call the second page using the 'after' value
+        String secondPageUrl = ORGANIZATION_MANAGEMENT_API_BASE_PATH + LIMIT_ARG + limit
+                + "&recursive=false&after=" + after;
+        Response secondPageResponse = getResponseOfGetWithOAuth2(secondPageUrl, m2mToken);
+
+        validateHttpStatusCode(secondPageResponse, HttpStatus.SC_OK);
+
+        int secondPageOrgCount = secondPageResponse.jsonPath().getList("organizations").size();
+        Assert.assertEquals(secondPageOrgCount, Math.min(limit, totalCreatedOrganizations - firstPageOrgCount));
+
+        List<Map<String, String>> secondPageLinks = secondPageResponse.jsonPath().getList("links");
+        before = getLink(secondPageLinks, "previous");
+        after = getLink(secondPageLinks, "next");
+
+        Assert.assertNotNull(before, "Before value should not be null on the second page.");
+        if (totalCreatedOrganizations > limit * 2) {
+            Assert.assertNotNull(after, "After value should not be null if there are more pages.");
+        } else {
+            Assert.assertNull(after, "After value should be null if this is the last page.");
+        }
+
+        // Step 3: Call the previous page using the 'before' value
+        String previousPageUrl = ORGANIZATION_MANAGEMENT_API_BASE_PATH + LIMIT_ARG + limit
+                + "&recursive=false&before=" + before;
+        Response previousPageResponse = getResponseOfGetWithOAuth2(previousPageUrl, m2mToken);
+
+        validateHttpStatusCode(previousPageResponse, HttpStatus.SC_OK);
+
+        int previousPageOrgCount = previousPageResponse.jsonPath().getList("organizations").size();
+        Assert.assertEquals(previousPageOrgCount, firstPageOrgCount);
+
+        // Ensure that the organizations match the first page organizations
+        Assert.assertEquals(previousPageResponse.jsonPath().getList("organizations"),
+                firstPageResponse.jsonPath().getList("organizations"),
+                "The previous page should return the same organizations as the first page.");
+
+        // Extract only the name and description from the previousPageResponse for comparison
+        List<Map<String, String>> actualOrganizations = previousPageResponse.jsonPath().getList("organizations");
+
+        List<Map<String, String>> actualOrganizationsSimplified = new ArrayList<>();
+        for (Map<String, String> org : actualOrganizations) {
+            Map<String, String> simplifiedOrg = new HashMap<>();
+            simplifiedOrg.put(ORGANIZATION_NAME, org.get(ORGANIZATION_NAME));
+            simplifiedOrg.put(ORGANIZATION_DESCRIPTION, "Test " + org.get(ORGANIZATION_NAME));
```

Comment:
When creating orgs, we are adding the description in the format of `Test Organization 1`. But here, for assertion, we are using the format `Test Org-1`. Check the correctness of this approach. 

If can, extract these string values to constants since they are used in multiple places.

Reply:
Done